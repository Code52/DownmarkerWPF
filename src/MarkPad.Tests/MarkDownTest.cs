using ApprovalTests;
using ApprovalTests.Reporters;
using MarkPad.Document;
using Xunit;

namespace MarkPad.Tests
{
	//[UseReporter(typeof(FileLauncherReporter))]
	public class MarkDownTest
	{
		[Fact]
		public void TestParsing()
		{
			var text = @"# This is a header block

With a Url  [HyperLink](http://code52.org/DownmarkerWPF/) 
and some **bold** formatting and *italics*

## And a subheader

 - With a bullet point 1
 - And bullet point 2
 
 Finally there is a
 
    code { block;} =  here
    
Of course we could do `code` in the middle of the page";
			var output = DocumentParser.Parse(text);
			Approvals.VerifyHtml(output);
		}
	}
}