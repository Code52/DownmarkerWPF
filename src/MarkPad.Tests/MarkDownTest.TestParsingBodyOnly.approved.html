<h1>This is a header block</h1>
<p>With a Url  <a href="http://code52.org/DownmarkerWPF/" target="_blank">HyperLink</a> 
and some <strong>bold</strong> formatting and <em>italics</em></p>
<h2>And a subheader</h2>
<ul>
<li>With a bullet point 1</li>
<li>And bullet point 2</li>
</ul>
<p>Finally there is a</p>
<pre><code>code { block;} =  here
</code></pre>

<p>Of course we could do <code>code</code> in the middle of the page</p>
